src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,46 @@
+.stopwatch {
+  position: relative;
+
+  @include circle($watchSize);
+  border: #{$watchBorderSize} solid $watchBorderColor;
+
+  background-color: $watchBgColor;
+
+  &::before,
+  &::after {
+    content: "";
+
+    @extend %absolute-centering;
+
+    width: $arrowsWidth;
+
+    border-radius: $arrowsBorderRadius;
+
+    transform-origin: bottom;
```

Comment:
> You can made a mixin and it will be more comfortable for brouser

I'm not sure this mixin could be reusable enough and as for me it makes readability of our code even worse